includes/constants/class-payment-request-button-states.php

```diff
@@ -0,0 +1,1162 @@
+			'RR' => [ 'RR', 'Roraima', NULL ],
+			'SC' => [ 'SC', 'Santa Catarina', NULL ],
+			'SP' => [ 'SP', 'São Paulo', NULL ],
+			'SE' => [ 'SE', 'Sergipe', NULL ],
+			'TO' => [ 'TO', 'Tocantins', NULL ],
+		],
+		// Canada.
+		'CA' => [
+			'AB' => [ 'AB', 'Alberta', 'Alberta' ],
+			'BC' => [ 'BC', 'British Columbia', 'Colombie-Britannique' ],
+			'MB' => [ 'MB', 'Manitoba', 'Manitoba' ],
+			'NB' => [ 'NB', 'New Brunswick', 'Nouveau-Brunswick' ],
+			'NL' => [ 'NL', 'Newfoundland and Labrador', 'Terre-Neuve-et-Labrador' ],
+			'NT' => [ 'NT', 'Northwest Territories', 'Territoires du Nord-Ouest' ],
+			'NS' => [ 'NS', 'Nova Scotia', 'Nouvelle-Écosse' ],
+			'NU' => [ 'NU', 'Nunavut', 'Nunavut' ],
+			'ON' => [ 'ON', 'Ontario', 'Ontario' ],
+			'PE' => [ 'PE', 'Prince Edward Island', 'Île-du-Prince-Édouard' ],
+			'QC' => [ 'QC', 'Quebec', 'Québec' ],
+			'SK' => [ 'SK', 'Saskatchewan', 'Saskatchewan' ],
+			'YT' => [ 'YT', 'Yukon', 'Yukon' ],
+		],
+		// Switzerland.
+		'CH' => [],
+		// China.
+		'CN' => [
+			'CN1'  => [ 'Yunnan Sheng', 'Yunnan Sheng', '云南省' ],
+			'CN2'  => [ 'Beijing Shi', 'Beijing Shi', '北京市' ],
+			'CN3'  => [ 'Tianjin Shi', 'Tianjin Shi', '天津市' ],
+			'CN4'  => [ 'Hebei Sheng', 'Hebei Sheng', '河北省' ],
+			'CN5'  => [ 'Shanxi Sheng', 'Shanxi Sheng', '山西省' ],
+			'CN6'  => [ 'Neimenggu Zizhiqu', 'Neimenggu Zizhiqu', '内蒙古' ],
+			'CN7'  => [ 'Liaoning Sheng', 'Liaoning Sheng', '辽宁省' ],
+			'CN8'  => [ 'Jilin Sheng', 'Jilin Sheng', '吉林省' ],
+			'CN9'  => [ 'Heilongjiang Sheng', 'Heilongjiang Sheng', '黑龙江省' ],
+			'CN10' => [ 'Shanghai Shi', 'Shanghai Shi', '上海市' ],
+			'CN11' => [ 'Jiangsu Sheng', 'Jiangsu Sheng', '江苏省' ],
+			'CN12' => [ 'Zhejiang Sheng', 'Zhejiang Sheng', '浙江省' ],
+			'CN13' => [ 'Anhui Sheng', 'Anhui Sheng', '安徽省' ],
+			'CN14' => [ 'Fujian Sheng', 'Fujian Sheng', '福建省' ],
+			'CN15' => [ 'Jiangxi Sheng', 'Jiangxi Sheng', '江西省' ],
+			'CN16' => [ 'Shandong Sheng', 'Shandong Sheng', '山东省' ],
+			'CN17' => [ 'Henan Sheng', 'Henan Sheng', '河南省' ],
+			'CN18' => [ 'Hubei Sheng', 'Hubei Sheng', '湖北省' ],
+			'CN19' => [ 'Hunan Sheng', 'Hunan Sheng', '湖南省' ],
+			'CN20' => [ 'Guangdong Sheng', 'Guangdong Sheng', '广东省' ],
+			'CN21' => [ 'Guangxi Zhuangzuzizhiqu', 'Guangxi Zhuangzuzizhiqu', '广西' ],
+			'CN22' => [ 'Hainan Sheng', 'Hainan Sheng', '海南省' ],
+			'CN23' => [ 'Chongqing Shi', 'Chongqing Shi', '重庆市' ],
+			'CN24' => [ 'Sichuan Sheng', 'Sichuan Sheng', '四川省' ],
+			'CN25' => [ 'Guizhou Sheng', 'Guizhou Sheng', '贵州省' ],
+			'CN26' => [ 'Shaanxi Sheng', 'Shaanxi Sheng', '陕西省' ],
+			'CN27' => [ 'Gansu Sheng', 'Gansu Sheng', '甘肃省' ],
+			'CN28' => [ 'Qinghai Sheng', 'Qinghai Sheng', '青海省' ],
+			'CN29' => [ 'Ningxia Huizuzizhiqu', 'Ningxia Huizuzizhiqu', '宁夏' ],
+			'CN30' => [ 'Macau', 'Macau', '澳门' ],
+			'CN31' => [ 'Xizang Zizhiqu', 'Xizang Zizhiqu', '西藏' ],
+			'CN32' => [ 'Xinjiang Weiwuerzizhiqu', 'Xinjiang Weiwuerzizhiqu', '新疆' ],
+			// [ 'Taiwan', 'Taiwan', '台湾' ],
```

Comment:
States that do not match with any WC state, or that are remaining, are commented out.